rating.cpp

```diff
@@ -0,0 +1,57 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <conio.h>
+int main(int newGameRes){
```

Comment:
The `main` function must be defined only once where the entrypoint is meant to be.